packages/language/src/validation/pli-validator.ts

```diff
@@ -273,4 +295,37 @@ export class Pl1Validator {
       }
     }
   }
+
+  checkImplicitBuiltins(
+    node: AST.ReferenceItem,
+    acceptor: ValidationAcceptor,
+  ): void {
+    // Skip if there is no process group information available.
+    if (!this.processGroup) {
+      return;
+    }
+
+    // Check for present name
+    if (!node.ref?.node?.name) {
+      return;
+    }
+
+    // Check if the reference item is a builtin.
+    if (node.ref?.node?.nameToken?.payload.uri?.scheme !== BuiltinsUriSchema) {
+      return;
+    }
+
+    const name = node.ref.node.name.toUpperCase();
+    if (!this.processGroup["implicit-builtins"]?.includes(name)) {
+      acceptor(
+        Severity.E,
+        PLICodes.Error.IBM1373I.message(node.ref?.node?.name),
```

Comment:
I believe we don't need the checks.
```suggestion
        PLICodes.Error.IBM1373I.message(node.ref.node.name),
```